internal/app/feed/feed.go

```diff
@@ -33,6 +33,8 @@ type Exporter interface {
 	CreateSchema(feed Feed, rows interface{}) error
 
 	WriteRows(feed Feed, rows interface{}) error
+	BulkUpdateRows(feed Feed, primaryKeys []string, element map[string]interface{}) (int64, error)
```

Comment:
All of these functions appear to operate in bulk, is the `Bulk` prefix needed?

Reply:
not 100%, will fix